tests/acl/test_acl.py

```diff
@@ -474,6 +478,21 @@ def direction(self, request):
         """Parametrize test based on direction of traffic."""
         return request.param
 
+    def check_rule_counters(self, duthost):
+        logger.info('Wait all rule counters are ready')
+
+        return wait_until(300, 2, self.check_rule_counters_internal, duthost)
```

Comment:
300 seems a bit long, could we set this to something like `30` or `60`?

Reply:
Updated code to wait for ACL rule counters ready in 60 seconds. Thanks.